src/api/requests.js

```diff
@@ -1,6 +1,82 @@
 import axios from "axios";
+import authHeader from "../services/auth-header";
 
-const server_url = "https://cello-exercise-index.herokuapp.com/"
+const server_url = "https://cello-exercise-index.herokuapp.com/";
+
+/** log into the system */
+export function adminLogin(loginInfo, setSuccess) {
+  axios
+    .post(`${server_url}api/token/`, loginInfo)
+    .then((response) => {
+      if (response.data.access) {
+        localStorage.setItem("admin", JSON.stringify(response.data));
+        setSuccess(true);
+      }
+
+      return response.data;
+    })
+    .catch((err) => {
+      console.log(err);
+      setSuccess(false);
+    });
+}
+
+/** get all the requests */
+export function getAllRequests(setRequests) {
+  axios.get(server_url + "api/requested/").then((res) => {
+    console.log(res.data.results);
```

Comment:
Remove console.logs

Reply:
already did in the recent comment!

Reply:
Do you mean you commited the changes locally but haven't pushed it yet?

Reply:
I've already pushed it, and it appears here:)
![Screen Shot 2022-12-10 at 1 51 05 PM](https://user-images.githubusercontent.com/83269618/206870823-1deec7b7-56cd-4fca-98a1-a9b766dc26f7.png)

Reply:
@eun-chae-s looks like you removed the console.log for getAllTags but some still exist in requests.js

Reply:
oohh let me check again